subsys/net/ip/ipv6.h

```diff
@@ -106,6 +106,11 @@ struct net_ipv6_nbr_data {
 
 	/** Is the neighbor a router */
 	bool is_router;
+
```

Comment:
Make sure that there are no holes in struct after that, it looks like we need to place the counter in correct position in this table.

Reply:
It looks like we can leave the counter at the end of the struct, there should not be any holes atm.